src/Result.ts

```diff
@@ -8,7 +8,6 @@ import { showStory } from './StorybookPage';
  * Violations reported by Axe for a story.
  */
 export type Result = {
-  name: string;
```

Comment:
The name of the story is now passed to the formatter in the various events. It's not needed on the result object anymore.